src/views/MyRequests/Table.tsx

```diff
@@ -28,13 +34,32 @@ const Table = () => {
 
   return (
     <ul>
-      {requests?.map(request => (
-        <li key={request.id}>
-          <UswdsLink asCustom={Link} to={`/508/requests/${request.id}`}>
-            {request.name}
-          </UswdsLink>
-        </li>
-      ))}
+      {requests?.map(request => {
+        let link: string;
+        switch (request.type) {
+          case RequestType.ACCESSIBILITY_REQUEST:
+            link = `/508/requests/${request.id}`;
+            break;
+          case RequestType.GOVERNANCE_REQUEST:
+            link = `/governance-task-list/${request.id}`;
+            break;
+          default:
```

Comment:
I'm wondering if we can get rid of the default here. Will TypeScript ensure that we are handling all of the types, or are things not setup that way?

Reply:
TypeScript is complaining that there's no default when I remove it. :sigh: